core/src/test/resources/log4j.properties

```diff
@@ -1,5 +1,7 @@
 log4j.rootLogger=INFO, stdout
 
+log4j.logger.gwi.s8.impl.PartitionedDagState$=WARN
+
 # Direct log messages to stdout
 log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.Target=System.out
```

Comment:
@pribor I'm wondering, what if there is multiple log4j.properties on classpath? Do the get merged ?

Reply:
I did the exact same thing in qpipe, but it still logs those graphs.

Reply:
The first log4j.properties is used and for tests this one has higher priority in the classpath...